docs/guides/api-rest/python-api.md

```diff
@@ -48,12 +48,12 @@ def handler(event, context):
   current_time = datetime.datetime.now().time()
 
   body = {
-      "message": "Hello, the current time is " + str(current_time)
+      'message': 'Hello, the current time is ' + str(current_time)
```

Comment:
Perhaps a format string, instead?

https://matthew-brett.github.io/teaching/string_formatting.html

Reply:
I'm not opposed, but, in that case I'd recommend reformatting all of the py docs (~10) with with either `black`/`autopep` (sep PR) to make it standard across examples along with any updated language constructs.